doc/source/parameters/dem/lagrangian_physical_properties.rst

```diff
@@ -0,0 +1,71 @@
+Lagrangian Physical Properties
+----------------------------------
+In this subsection, the physical properties of the simulation are defined. These properties include gravitational acceleration (``gx``, ``gy`` and ``gz``), ``number of particle types``, and for each type, particle ``diameter``, particle ``density``, ``Young's modulus`` of particle and wall, ``Poisson ratio`` of particle and wall, ``restitution coefficient`` of particle and wall, ``friction coefficient`` of particle and wall and ``rolling friction coefficient`` of particle and wall.
+
+.. note::
+    If the particles in the simulation are monodispersed, the ``number of particle types`` should be equal to zero. For polydispersed systems, the ``number of particle types`` is selected equal to the number of particles types in the simulation. For each particle type, a separate subsection ``particle type n`` should be defined (n starts from zero to ``number of particle types`` - 1) which contains all the physical properties related to that particle type.
+
+.. note::
+    For each particle type, two ``size distribution type``s can be defined: ``uniform`` and ``normal``. In ``uniform`` size distribution, the diameter of the particles is constant, while in ``normal`` size distribution, the particle diameters are sampled from a normal distribution with an average of ``average diameter`` and standard deviation of ``standard deviation``.
+
+.. code-block:: text
+
+ subsection model parameters
+  # Gravitational acceleration in x direction
+  set gx            		 			 = 0.0
+
+  # Gravitational acceleration in y direction
+  set gy            		 			 = 0.0
+
+  # Gravitational acceleration in z direction
+  set gz            		 			 = -9.81
```

Comment:
I really think gravity should become it's own section eventually. It's not a physical property...

Reply:
We discussed it once with @Luckabarbeau. Maybe we can define a unified gravity in Lethe.